LbhNCCApi/Actions/UHActions.cs

```diff
@@ -272,5 +272,25 @@ public List<TenancyTransactionStatements> GetAllTenancyTransactionStatements(str
             return lstTransactionsState;
 
         }
+
+        public async Task UpdateRecordingDetails( int actionDiaryId, DateTime updateDate)
+        {
+            using (var conn = new SqlConnection(_uhtconnstring))
+            {
+                SqlCommand cmd = null;
+               
+                    cmd = new SqlCommand("UPDATE araction SET action_date=@action_date" +
+                                                    " WHERE araction_sid=@Id", conn);
+
+                conn.Open();
```

Comment:
Hi Shweta.  I think that maybe we should also close the connection after we execute the query.